### PR TITLE
Cosmos delegator info

### PR DESCRIPTION
**Pull request by Pantani**
need merge #397

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,4 +24,5 @@ require (
 	github.com/ybbus/jsonrpc v2.1.2+incompatible
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
 	golang.org/x/net v0.0.0-20190620200207-3b0461eec859 // indirect
+	gopkg.in/yaml.v2 v2.2.2
```
Comment (vikmeup) [inline go.mod]: is this something new?

File: platform/cosmos/api.go [hunk 2]
```diff
@@ -101,19 +101,30 @@ func (p *Platform) GetDelegations(address string) (blockatlas.DelegationsPage, e
...
 }
 
-func NormalizeDelegations(delegations []Delegation) []blockatlas.Delegation {
+func NormalizeDelegations(delegations []Delegation, validators blockatlas.ValidatorMap) []blockatlas.Delegation {
 	c := coin.Cosmos()
 	results := make([]blockatlas.Delegation, 0)
 	for _, v := range delegations {
+		validator, ok := validators[v.ValidatorAddress]
+		if !ok {
+			logger.Error("Validator not found", validator)
```
Comment (vikmeup) [inline platform/cosmos/api.go]: do you need to continue?

Comment (Pantani) [reply platform/cosmos/api.go]: You know more than me, can you help me with this question? Do we need to continue? I think we don't have a problem. We can show just the JSON below in this case.

```
[
   {
      "delegator":{
         "id":"cosmosvaloper1qwl879nx9t6kef4supyazayf7vjhennyh568ys",
      },
      "coin":{
         "coin":118,
         "symbol":"ATOM",
         "name":"Cosmos ATOM",
         "decimals":6
      },
      "value":"3000000",
      "status":"active"
   }
]
```

Comment (vikmeup) [reply platform/cosmos/api.go]: i don't think it make sense to return this way. if validated not found with information we should not present it here.

Comment (vikmeup) [reply platform/cosmos/api.go]: we should only present validators that provided information in assets repo

Comment (Pantani) [reply platform/cosmos/api.go]: great. I going to fix this. Thks

**Pull request merged**